doximity/orientation
# Algolia integration

Tracker story link: https://www.pivotaltracker.com/story/show/144892879

Main goals:
1 - better matching
2 - autosuggest
3 - show which sections/parts of the page matched
4 - be able to customize how we score title vs body

QA Instructions:
I've deployed this branch to: https://dox-orientation-qa.herokuapp.com/
As far as testing the goals: go the 'Articles' tab and for each goal, do the following:
1 - search something in the search bar, you should get more relevant results verses our previous PSQL text search, e.g. type in 'onboarding'
2 - start typing something, you should start to see suggestions of articles that correspond to what you're typing
3 - as you type something, that part should be bolded in the suggestions that pop up
4 - this isn't really testable, but titles are set to be more important than body, so if you type a word (e.g. 'Gemfury') that appears in the title of article 1 and then also shows up in the body of article 2, article 1 should be higher up in suggestions/results
